docs/layouts/models/single.html

```
@@ -18,12 +34,9 @@ <h3>
             {{ if .name }}
             <div class="maincontainer">
                 <div class="componentimg">
-                    <img id="logo-dark-light"
-                        src="{{ strings.TrimPrefix "/" .colorIcon | relURL }}"
-                        alt="{{ .name }}"
-                        onerror="handleImageError(this)"
-                        data-logo-for-dark="{{ strings.TrimPrefix "/" .whiteIcon | relURL }}"
-                        data-logo-for-light="{{ strings.TrimPrefix "/" .colorIcon | relURL }}" />
+                    <img id="logo-dark-light" src="{{ strings.TrimPrefix " /" .colorIcon | relURL }}" alt="{{ .name }}"
```

HTMLHint (1.9.2)

[error] Special characters must be escaped : [ < ].

(spec-char-escape)